cli-requirejs-demo/aurelia_project/aurelia.json

```diff
@@ -105,15 +99,6 @@
               "css/bootstrap.css"
             ]
           },
-          {
-            "name": "flatpickr",
-            "path": "../node_modules/flatpickr/dist",
-            "main": "flatpickr.min",
-            "resources": [
-              "flatpickr.min.css",
-              "l10n/fr.js"
-            ]
-          },
           {
```

Comment:
I have no idea where this alias is actually requested, guess internally, but in order to keep things working this is needed as well

Reply:
Some of the 3rd party libs I use with Aurelia-SlickGrid are Flatpickr, multiple-select.js and SlickGrid itselfs requires jQuery + jQueryUI. Does that help?

Reply:
Yeah all fine, i just figured you ship multi-select by yourself. Anyways just wanted to note why still some manual config is needed, especially for BS3

Reply:
ah not sure about BS3, could it be just for the `$` alias, but if you want to update the demo to BS4 you can do that too does it require the manual config with BS4? I'll give it a try tonight after work and probably merge at that time

Reply:
Except for the jquery-ui-dist part nope, I don't think so. I merely updated the existing requirejs sample, which used to be BS3. Would you prefer another one created for BS4 so that both co-exist for requirejs? Also note that SystemJS is no longer supported with the latest CLI but instead Alameda is.

Reply:
I saw Alameda in the CLI, that was the first time I ever seen this name, yeah I'll remove SystemJS (someone asked for it some time ago and it looks like that person still uses it as shown in this [issue](https://github.com/ghiscoding/aurelia-slickgrid/issues/364)).

If you have time to change it to BS4 or create another one for BS4, that would be great, else I can look at it later. I Personally use WebPack everywhere. Also it was BS3 when I started the lib and I saw some small little styling differences with BS3 vs BS4, it might need some CSS adjustments in some areas.

Reply:
perhaps in order to stay aligned with the webpack samples it might make sense to keep BS3 and BS4 around the same time. I'll update this PR so keep it open for a bit longer